lib/embed/viewer/media.rb

```diff
@@ -39,6 +39,13 @@ def self.supported_types
         [:media]
       end
 
+      # override CommonViewer instance method to ensure we do not show download panel when no downloadable files
+      def show_download?
+        show_it = method(:show_download?).super_method.call
+        return @purl_object.downloadable_files.present? if @purl_object && show_it
+        show_it
```

Comment:
So, I admit I don't quite understand what we're trying to accomplish here.

Can line 44 just be `show_it = super`?

On line 45, I'm not sure we will ever have the context of the `@purl_object` not existing.  That gets set in the initializer for this class, so I don't know if we need to check that here.